CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+## [0.1.0] - not released yet
```

Comment:
It looks like in the "Keep a Changelog" site, they suggest making this
```suggestion
## [Unreleased]
```
and then moving it to a version once you make a release